can/io/logger.py

```diff
@@ -105,6 +107,10 @@ def compress(
         File will automatically recompress upon close.
         """
         real_suffix = pathlib.Path(filename).suffixes[-2].lower()
+        if real_suffix in Logger.incompatible_gzip:
```

Comment:
```suggestion
        if real_suffix in (".blf", ".db"):
```

I would suggest to remove the class attribute until someone complains about the missing access to it.

Reply:
Done.